### ISSUE TITLE
[STANDUP] -- 2019-08-07

### ISSUE THREAD

**Issue by nickcannariato** [nickcannariato/erin-nick#3]
Good morning all!

Respond to this issue by answering the following questions:

1. What did you plan to get done yesterday?
2. Did you accomplish your goals yesterday?
3. Why/Why not?
4. What do you plan to get done today?

**Comment by nickcannariato** [comment]
> 1. What did you plan to get done yesterday?

- [ ] Finish GraphQL Course
- [x] Get portfolio site instantiated & deployed (at minimum)
- [x] Start MongoDB University
- [x] Start next set of Algo questions

> 2. Did you accomplish your goals yesterday?

Mostly yes. I didn't finish (or even attempt to finish) the GraphQL course, but I got my portfolio site set up with a full testing pipeline and deployed, which was most important. I also kept working on Algo stuff and got MongoDB University done for this week.

> 3. Why/Why not?

Yesterday wound up being a lot of doctor visits with Michelle. I was still able to get a lot of work done from the hospital but there are limitations for sure.

> 4. What do you plan to get done today?

- [ ] Keep moving forward on GraphQL
- [ ] Keep working on my portfolio site
    - [ ] Decide on the stateful design of the components I'll be using
    - [ ] Build out boilerplate for the sections of the page
    - [ ] Read up a bit more on TailwindCSS so I can use it effectively in the project
- [ ] Apply for Thorn

**Comment by erin-koen** [comment]
> 1. What did you plan to get done yesterday?
- [ ]  Egghead Advanced React Patterns [https://egghead.io/courses/advanced-react-component-patterns]
- [ ]   Django Polls App
- [ ]   Start Udemy Django API course
- [ ]   Refill jobs queue: Find 5 Jobs to which to apply tomorrow.

> 2. Did you accomplish your goals yesterday?

*Django polls app*: I got my first route wired up. I would've been further along, but had a setup issue where I'd reversed the name of my project and my app. I fixed it naively, by just changing folder names, but that did more harm than good. So I burned the whole thign down and started over again. Good learning expereince

*Start Udemy Django API course*: contingent on the above, so no progress here.

*Refill jobs queue*: I did get a bunch of jobs added to the queue, applied to a couple more, and followed up with some networking on a few more. Full details on hubspot pipeline.

*Egghead Advanced React Patterns*: I got the lessons in that I felt i needed to prior to my Skilled interview this morning based on feedback I'd gotten from people who had already done theirs. 

> 3. Why/Why not?
Mostly successful yesterday, hopefully I can cruise through Django today. 

> 4. What do you plan to get done today?
- [ ] Finish Django polls
- [ ] Start Udemy Django API course
- [ ] Follow up on outstanding networking items (logged to hubspot)
- [ ] Apply to jobs in queue
     - [ ] Bloomberg - finish cover letter
     - [ ] Plaid - follow up with referral
     - [ ] Gemini - chase down referral
     - [ ] Etsty - finish cover letter
